tests/common/value_helper.h

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2020-2022 The Khronos Group Inc.
+//  Copyright:	(c) 2017 by Codeplay Software LTD. All Rights Reserved.
+//
+//  This file contains helper functions for modifying and comparing values,
+//  arrays, and objects that implements operator[]
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_COMMON_VALUE_HELPER_H
+#define __SYCLCTS_TESTS_COMMON_VALUE_HELPER_H
+#include "../../util/type_traits.h"
+
+#include <cassert>
+
+namespace value_helper {
+
+template <typename T, size_t N>
+using ArrayT = T[N];
+
+// Modify functions
+template <typename T, size_t N>
+inline void change_val(ArrayT<T, N>& left, const T& right) {
```

Comment:
Maybe call these functions `assign` instead?

Reply:
I think it makes more sense. I changed the name to `assign`